.drone.yml

```diff
@@ -1,351 +1,344 @@
     when:
       repo: vmware/vic
-      event: [ push, tag ]
-      branch: [ refs/tags/*, releases/* ]
+      event: [push, tag]
+      branch: ['refs/tags/*', 'releases/*']
       status: success
 
   trigger-downstream:
-    image: harbor.ci.drone.local/library/vic-downstream-trigger:1.0
+    image: 'gcr.io/eminent-nation-87317/vic-downstream-trigger:1.0'
     environment:
-      DRONE_SERVER: ${DRONE_SERVER}
-      DRONE_TOKEN: ${DRONE_TOKEN}
       SHELL: /bin/bash
+    secrets:
+      - drone_server
+      - drone_token
     when:
       repo: vmware/vic
-      event: [ push, tag ]
-      branch: [ master, releases/*, refs/tags/* ]
+      event: [push, tag]
+      branch: [master, 'releases/*', 'refs/tags/*']
       status: success
 
   report-coverage:
-     image: robertstettner/drone-codecov
-     token: ${CODECOV_TOKEN}
-     files:
-       - .cover/cover.out
+    image: robertstettner/drone-codecov
+    secrets:
+      - codecov_token
+    files:
+      - .cover/cover.out
 
   notify-slack-on-fail:
     image: plugins/slack
-    webhook: ${SLACK_URL}
+    secrets:
+      - source: slack_url
+        target: slack_webhook
     username: drone
-    template: >
-      Build https://ci.vcna.io/vmware/vic/{{ build.number }} by {{ build.author }} finished with a {{ build.status }} status. Logs: https://console.cloud.google.com/m/cloudstorage/b/vic-ci-logs/o/integration_logs_{{ build.number }}_{{ build.commit }}.zip?authuser=1
+    template: "Build https://ci.vcna.io/vmware/vic/{{ build.number }} by {{ build.author }} finished with a {{ build.status }} status. Logs: https://console.cloud.google.com/m/cloudstorage/b/vic-ci-logs/o/integration_logs_{{ build.number }}_{{ build.commit }}.zip?authuser=1\n"
     when:
       repo: vmware/vic
-      branch: [ master, releases/*, refs/tags/* ]
+      event: [push, tag, deployment]
+      branch: [master, 'releases/*', 'refs/tags/*']
       status: failure
 
   notify-slack-on-pass:
     image: plugins/slack
-    webhook: ${SLACK_URL}
+    secrets:
+      - source: slack_url
+        target: slack_webhook
     username: drone
-    template: >
-      Build https://ci.vcna.io/vmware/vic/{{ build.number }} by {{ build.author }} finished with a {{ build.status }} status, find the build at: https://storage.googleapis.com/vic-engine-builds/vic_{{ build.number }}.tar.gz
+    template: "Build https://ci.vcna.io/vmware/vic/{{ build.number }} by {{ build.author }} finished with a {{ build.status }} status, find the build at: https://storage.googleapis.com/vic-engine-builds/vic_{{ build.number }}.tar.gz\n"
```

Comment:
Is this still ci.vcna.io? or is it ci-vic.vmware.com?

Reply:
I have PR open for that, will merge it in few min.

Reply:
Done